src/ios-deploy/ios-deploy.m

```diff
@@ -611,11 +614,17 @@ mach_error_t install_callback(CFDictionaryRef dict, int arg) {
     CFNumberGetValue(CFDictionaryGetValue(dict, CFSTR("PercentComplete")), kCFNumberSInt32Type, &percent);
 
     int overall_percent = (percent / 2) + 50;
-    NSLogOut(@"[%3d%%] %@", overall_percent, status);
+
+    CFStringRef path = CFDictionaryGetValue(dict, CFSTR("Path"));
+    NSString *status_with_path = path != NULL ?
```

Comment:
We should add a check that the user requested incremental installs and get the status only then to make sure we aren't changing this output in other cases.

Reply:
Done.